admin_tests/spam/test_views.py

```diff
@@ -177,7 +179,8 @@ def setUp(self):
         self.view = setup_form_view(self.view, self.request, self.form,
                                     spam_id=self.comment._id)
 
-    def test_get_context_data(self):
+    @mock.patch('admin.spam.views.render')
```

Comment:
It prevents it from calling the render which goes out of its way. I believe in the desk PR #5056 I remove this, this was to fix an issue with the tests.